scripts/import/import_EPMC.pl

```diff
@@ -526,56 +578,85 @@ sub report_summary{
                                          where p1.pmid = p2.pmid
                                          and p1.publication_id < p2.publication_id
                                          and p1.pmid is not null
-                                       ]);
+                                         ]);
 
+    # Query is taking a few hours to run
     my $dup2_ext_sth = $dba->dbc->prepare(qq[ select p1.publication_id, p2.publication_id, p2.pmcid
                                          from publication p1, publication p2
                                          where p1.pmcid = p2.pmcid
                                          and p1.publication_id < p2.publication_id
-                                       ]);
+                                         and p1.pmcid is not null
+                                         ]);
+
+   # my $dup2_ext_sth = $dba->dbc->prepare(qq[ SELECT publication_id, pmcid 
+   #                                           FROM publication 
+   #                                           GROUP BY pmcid,publication_id HAVING COUNT(*) > 1 ]);
```

Comment:
I assume this can be removed?

Reply:
I was testing if I could use that one but I don't get the output I want. Deleted.